README.md

```diff
@@ -20,11 +26,148 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+`ActiveService` defines the concept of `operation`. An operation is a method that will run some business function, for example creating a user. ActiveServices don't have state.
+
+### operation
+
+An example UsersService could look like this:
+
+```
+class UsersService
+	include ActiveService
+
+	# Ruby 2.1 required parameters :)
+	operation :create_user do |name:, email:|
+		user = User.create!(name: name, email: email)
+
+		UserMailer.welcome(user).deliver_later
+
+		user
+	end
+
+end
+```
+
+You can use this service like this:
+
+```
+UsersService.create_user(name: 'Jhon', email: 'jhon@x.com')
+```
+
+### pipe
+
+This service can be refactored to something like this:
```

Comment:
this wouldn't be an exact refactor right?
I mean, the returned value would be a `Mail` instead of a `User` like the example above right?